Brokerages/Bitfinex/BitfinexSymbolMapper.cs

```diff
@@ -0,0 +1,210 @@
+                throw new ArgumentException($"Invalid Bitfinex symbol: {brokerageSymbol}");
+
+            if (!IsKnownBrokerageSymbol(brokerageSymbol))
+                throw new ArgumentException($"Unknown Bitfinex symbol: {brokerageSymbol}");
+
+            if (securityType != SecurityType.Crypto)
+                throw new ArgumentException($"Invalid security type: {securityType}");
+
+            if (market != Market.Bitfinex)
+                throw new ArgumentException($"Invalid market: {market}");
+
+            return Symbol.Create(ConvertBitfinexSymbolToLeanSymbol(brokerageSymbol), GetBrokerageSecurityType(brokerageSymbol), Market.Bitfinex);
+        }
+
+        /// <summary>
+        /// Converts an Bitfinex symbol to a Lean symbol instance
+        /// </summary>
+        /// <param name="brokerageSymbol">The Bitfinex symbol</param>
+        /// <returns>A new Lean Symbol instance</returns>
+        public Symbol GetLeanSymbol(string brokerageSymbol)
+        {
+            var securityType = GetBrokerageSecurityType(brokerageSymbol);
+            return GetLeanSymbol(brokerageSymbol, securityType, Market.Bitfinex);
+        }
+
+        /// <summary>
+        /// Returns the security type for an Bitfinex symbol
+        /// </summary>
+        /// <param name="brokerageSymbol">The Bitfinex symbol</param>
+        /// <returns>The security type</returns>
+        public SecurityType GetBrokerageSecurityType(string brokerageSymbol)
+        {
+            if (string.IsNullOrWhiteSpace(brokerageSymbol))
+                throw new ArgumentException($"Invalid Bitfinex symbol: {brokerageSymbol}");
+
+            if (!IsKnownBrokerageSymbol(brokerageSymbol))
+                throw new ArgumentException($"Unknown Bitfinex symbol: {brokerageSymbol}");
+
+            return SecurityType.Crypto;
+        }
+
+        /// <summary>
+        /// Returns the security type for a Lean symbol
+        /// </summary>
+        /// <param name="leanSymbol">The Lean symbol</param>
+        /// <returns>The security type</returns>
+        public SecurityType GetLeanSecurityType(string leanSymbol)
+        {
+            return GetBrokerageSecurityType(ConvertLeanSymbolToBitfinexSymbol(leanSymbol));
+        }
+
+        /// <summary>
+        /// Checks if the symbol is supported by Bitfinex
+        /// </summary>
+        /// <param name="brokerageSymbol">The Bitfinex symbol</param>
+        /// <returns>True if Bitfinex supports the symbol</returns>
+        public bool IsKnownBrokerageSymbol(string brokerageSymbol)
+        {
+            if (string.IsNullOrWhiteSpace(brokerageSymbol))
+                return false;
+
+            return KnownSymbolStrings.Contains(brokerageSymbol);
+        }
+
+        /// <summary>
+        /// Checks if the currency is supported by Bitfinex
+        /// </summary>
+        /// <returns>True if Bitfinex supports the currency</returns>
+        public bool IsKnownBrokerageCurrency(string currency)
```

Comment:
Nice idea adding the list of currencies to the symbol mapper. 
Only comment is on the naming of the method, since `IsKnownBrokerageCurrency` could lead people to think it includes also cryptos, perhaps could be renamed to `IsKnownFiatCurrency` ?

Reply:
Okay, will rename